### PR TITLE
[FIX] Composer component NotAvailableOnCall

### PR DESCRIPTION
**Pull request by tiagoevanp**

### CODE REVIEW

File: client/components/voip/composer/NotAvailableOnCall/NotAvailableOnCall.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Box } from '@rocket.chat/fuselage';
+import React, { ReactElement } from 'react';
+
+import NotAvailableContent from './NotAvailableContent';
+import NotAvailableContentWrapper from './NotAvailableContentWrapper';
+
+const NotAvailableOnCall = (): ReactElement => (
+	<Box h='44px' w='100%' borderColor='disabled' borderWidth='2px'>
+		<NotAvailableContentWrapper>
+			<NotAvailableContent icon='message' text='Composer_not_available_phone_calls' />
```
Comment (MartinSchoeler) [inline client/components/voip/composer/NotAvailableOnCall/NotAvailableOnCall.tsx:10]: `text` needs translation

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]

**Review by MartinSchoeler — APPROVED** [review]

File: client/components/voip/composer/NotAvailableOnCall/NotAvailable.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export const Default: Story = () => (
+	<NotAvailableContentWrapper>
+		<NotAvailableContent text='Messages are not available on phone calls' icon='message' />
+	</NotAvailableContentWrapper>
+);
+
+export default {
+	title: 'components/Composer/NotAvailableOnCall',
+	component: Default,
```
Comment (tassoevan) [inline client/components/voip/composer/NotAvailableOnCall/NotAvailable.stories.tsx:15]: It should be `NotAvailableOnCall` here.

File: client/components/voip/composer/NotAvailableOnCall/NotAvailable.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Story } from '@storybook/react';
+import React from 'react';
+
+import NotAvailableContent from './NotAvailableContent';
+import NotAvailableContentWrapper from './NotAvailableContentWrapper';
+
+export const Default: Story = () => (
+	<NotAvailableContentWrapper>
+		<NotAvailableContent text='Messages are not available on phone calls' icon='message' />
+	</NotAvailableContentWrapper>
```
Comment (tassoevan) [inline client/components/voip/composer/NotAvailableOnCall/NotAvailable.stories.tsx:10]: Why not use `NotAvailableOnCall` directly here?

**Review by tassoevan — CHANGES_REQUESTED** [review]

File: client/components/voip/composer/NotAvailableOnCall/NotAvailable.stories.tsx (hunk 3 above)
Comment (tiagoevanp) [reply client/components/voip/composer/NotAvailableOnCall/NotAvailable.stories.tsx:10]: this component have useTranslation on it...

File: client/components/voip/composer/NotAvailableOnCall/NotAvailable.stories.tsx (hunk 2 above)
Comment (tiagoevanp) [reply client/components/voip/composer/NotAvailableOnCall/NotAvailable.stories.tsx:15]: this component has useTranslation on it...

Comment (tassoevan) [reply client/components/voip/composer/NotAvailableOnCall/NotAvailable.stories.tsx:15]: It's fine, it uses a mocked version of `TranslationProvider`.

**Review by tassoevan — APPROVED** [review]

**Pull request closed without merging**